modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/data/handler/BatchEnginePortletDataHandler.java

```diff
@@ -480,6 +500,24 @@ public boolean isPersist() {
 			});
 	}
 
+	private List<Registration> _getActiveRegistrations(
```

Comment:
I think this could be simplified by using the `ListUtil.filter` method

```java
	private List<Registration> _getActiveRegistrations(
		PortletDataContext portletDataContext) {

		return ListUtil.filter(
			_registrations,
			registration -> {
				ExportImportVulcanBatchEngineTaskItemDelegate.
					ExportImportDescriptor exportImportDescriptor =
						registration.getExportImportDescriptor();

				return exportImportDescriptor.isActive(portletDataContext);
			});
	}
```